fetch.bs

```diff
@@ -7520,6 +7542,25 @@ are:
  <li><p>Return <var>responseObject</var>.
 </ol>
 
+
+<p>The static
+<dfn method for=Response><code>json(<var>data</var>, <var>init</var>)</code></dfn> method steps
+are:
+
+<ol>
+ <li><p>Let <var>bytes</var> the result of running <a>serialize a JavaScript value to JSON bytes</a>
```

Comment:
This can fail (e.g., when a getter throws an exception), what do we want to do in that case? Do we want to throw an exception?

Reply:
Yes. From what I can tell the `JSON.stringify` call in `serialize a JavaScript value to a JSON string` will throw an exception in this case, which should just be propagated up.

Reply:
I'll add a test for this to the WPT.

Reply:
https://github.com/web-platform-tests/wpt/pull/32825/commits/004ac56823ea7b19160a9a97f9722bdd23bb7726